src/services/sessionInvite.js

```diff
@@ -213,7 +214,7 @@ module.exports = class UserInviteHelper {
 					},
 				}
 
-				parsedCSVData.push({
+				const parsedRow = {
```

Comment:
@sumanvpacewisdom what is the change here ?

Reply:
@rakeshSgr Intially we were hardcoding the status csv with what ever we had , but now we can add entity types and for certain organization , so i have made it dynamically